plugins/exporter.koplugin/main.lua

```diff
@@ -177,13 +177,23 @@ function Exporter:exportClippings(clippings)
     local export_callback = function()
         UIManager:nextTick(function()
             local timestamp = os.time()
+            local statuses = {}
             for k, v in pairs(self.targets) do
                 if v:isEnabled() then
                     v.timestamp = timestamp
-                    v:export(exportables)
+                    local status = v:export(exportables)
+                    if status then
+                        table.insert(statuses, _(v.name .. ": Exported to " ) .. v:getFilePath(exportables))
```

Comment:
This is going to crash on remote exporters.

Reply:
They don't have a getFilePath?

Reply:
yup. It returns nil

Reply:
Fixed it here: #9167